Assets/_BForBoss/_Core/Scripts/Boss/DerekContextManager.cs

```diff
@@ -0,0 +1,175 @@
+using System;
+using System.Collections;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class DerekContextManager : ContextManager
+    {
+        [SerializeField, Tooltip("How long should the system wait before restarting the Labor after an incomplete Labor")]
+        private float _LaborIncompleteTimer = 5.0f;
```

Comment:
typo -  _laborIn...

Reply:
good catch! At work, Unreal coding convention uses PascalCase for variables instead of camelCase, so my brain has been kinda scrambled haha